core/state_connector.go

```diff
@@ -187,10 +193,61 @@ func envAttestors(key string) []common.Address {
 	return attestors
 }
 
-func attestationSelector() []byte {
-	return []byte{0x29, 0xbe, 0x4d, 0xb2}
+// unused 'blockTime' might be used for hard forks in the future.
+func stateConnectorContract(chainID *big.Int, blockTime *big.Int) common.Address {
```

Comment:
If it's unused I'd personally rather not have it now, and add it the day we need it. WDYT?